server/app.js

```diff
@@ -45,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/notification", notificationRouter);
```

Comment:
It makes sense to name routes that access collections of resources with a plural name. 
```suggestion
app.use("/notifications", notificationRouter);
```
See [this article](https://restfulapi.net/resource-naming/) for more common naming conventions in REST.